bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -323,12 +323,22 @@ def VecOps(self):
     # Overload RDF namespace
     @property
     def RDF(self):
+        import numpy
```

Comment:
Now that I take another look at this, maybe we could move this import directly inside the `MakeNumpyDataFrameCopy` function so it's closest to the point of usage